inc/authldap.class.php

```diff
@@ -2802,8 +2802,10 @@ static function showUserImportForm(AuthLDAP $authldap) {
                   echo "<td>$label</td><td>";
                   $field_counter++;
                   echo "<input type='text' name='criterias[$field]' value='".
-                        (isset($_SESSION['ldap_import']['criterias'][$field])
-                         ?$_SESSION['ldap_import']['criterias'][$field]:'')."'>";
+                  if (isset($_SESSION['ldap_import']['criterias'][$field])) {
```

Comment:
if not set, ``$field_value`` will not be set also, and notice will triggers on 2808